CMakeLists.txt

```diff
@@ -2786,6 +2792,12 @@ IF(XNNPACK_BUILD_TESTS)
   TARGET_LINK_LIBRARIES(qd8-f32-qc4w-gemm-minmax-test PRIVATE gemm-microkernel-tester hardware-config logging microkernels-all microparams-init)
   ADD_TEST(NAME qd8-f32-qc4w-gemm-minmax-test COMMAND qd8-f32-qc4w-gemm-minmax-test)
 
+  ADD_EXECUTABLE(qd8-f32-qb4w-gemm-minmax-test test/qd8-f32-qb4w-gemm-minmax.cc)
```

Comment:
This is a leftover that I missed when cleaning up. I need to remove it from the PR.